SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
+
+## Reporting a Vulnerability
+
+Use this section to tell people how to report a vulnerability.
+
+Tell them where to go, how often they can expect to get an update on a
```

Comment:
I do not think we have a policy for reporting a vulnerability. 
Papa is just parsing CSV so normally there is no way to exploit a CSV parser. 
For now, I think we can still use github issues for vulnerability reporting as we do not have any better alternative.